### ISSUE TITLE
feat: sync saves and games between MiSTer and Analogue Pocket

### ISSUE THREAD

**Issue by mikeyvids** [wizzomafizzo/mrext#91]
Support transferring data between MiSTer and Analogue Pocket in both directions.

Requested workflows:

- Synchronize compatible game saves, with safe handling when both copies changed
- Migrate or synchronize ROM libraries between platform-specific folder layouts
- Extract MiSTer ZIP libraries when Pocket cores require unpacked ROMs
- Optionally limit game transfer to user-managed `Sync` folders or recently played titles
- Work with the Pocket SD card mounted through a card reader rather than relying on slow USB transfer

**Comment by wizzomafizzo** [comment]
I would have liked to add this too but I'm not sure it's possible right no over the USB cable connection, because it is so crazy slow. You would need to use a SD card reader in the mister. Is this still useful that way?

**Comment by mikeyvids** [comment]
Yes, having to eject MicroSD from the Pocket is needed to perform basic core updates in a timely manner with Pupdate and Pocket Sync. It is a perfectly reasonable expectation to do the same for this use case. 

Regarding library sync, having recently updated both mister and pocket libraries a significant issue may be that all mister libraries would have to be unzipped, since Pocket does not read zip libraries. 

Giving it some thought most of the games that I want to be synced would be ones I recently played or those I drop in recently with intent to play. Rather than updating the entire library a quicker solution may be having a dedicated Sync folder in each games library, where it ONLY syncs the contents of this folder in each core library e.g. a handful of new romhacks you are playing or want to take on the go / vice versa. This would cutdown on the sync transfer time and file size for transfer. The Sync folder could be something managed by an end user.

Another method I thought has a prerequisite - Does MiSTer track recently played titles? If so copying a file path from a recently played title to that sync folder could be a neat feature (say a limited lookup of the past 5-10 games, copy the file path from the source to the destination; maybe bypass the need for sync folder.) Drawback to this is you go MiSTer to Pocket, unless Pocket has a similar way to track recently played titles for developers.

**Comment by wizzomafizzo** [comment]
Pocket-specific utilities have been retired from mrext. Safe bidirectional synchronization would require device discovery, core-specific path and format mapping, archive conversion, conflict resolution, and strong protection against data loss, so a new MiSTer/Pocket sync system is not planned here.

Pocket Sync already provides maintained MiSTer save mapping and is the appropriate option for the save-sync portion:
https://github.com/neil-morrison44/pocket-sync

That recommendation does not claim full support for the requested bidirectional ROM-library workflow, which mrext does not provide.
